tests/utils/archives.py

```diff
@@ -14,8 +14,9 @@
 import zipfile
 from typing import List
 
+from archive_path import read_file_in_tar, read_file_in_zip
```

Comment:
just a minor nitpick: I'd probably call it `read_file_from_tar` (but not necessary to change)

Reply:
again an archive-path matter 😄

Reply:
> again an archive-path matter 😄

Correct me if I'm wrong but at least the API of the package should remain part of this review (probably the code should as well).

In any case, this particular suggestion is very minor and not important.